torch/_dynamo/output_graph.py

```diff
@@ -633,8 +633,9 @@ def bind_symint(s, prop):
         def handle_tensor(t, src):
             for i, s in enumerate(t.size()):
                 bind_symint(s, TensorPropertySource(src, TensorProperty.SIZE, i))
-            for i, s in enumerate(t.stride()):
-                bind_symint(s, TensorPropertySource(src, TensorProperty.STRIDE, i))
+            if t.layout is torch.strided:
```

Comment:
This if-block should capture `storage_offset` as well because `storage_offset` is stride layout specific property.

Reply:
I will leave this fix than for a later PR when it is easier to test